### PR TITLE
Ccs 3743 change module type fixes

### PR DESCRIPTION
**Pull request by xdavidson**
This PR updates the label in the sort filter. It changes the label from  "Module type" to "Content type" so that it matches the column name.

### CODE REVIEW

File: pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap [hunk 1]
```diff
@@ -152,7 +152,7 @@ exports[`SearchFilter tests should render default Search component 1`] = `
       <FormSelectOption
         isDisabled={false}
         key="Module type"
-        label="Module type"
+        label="Content type"
         required={false}
         value="Module type"
```
Comment (rednitish) [inline pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap:157]: Can we also consider changing other module instances changes within this PR?

File: pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap [hunk 2]
```diff
@@ -152,7 +152,7 @@ exports[`SearchFilter tests should render default Search component 1`] = `
       <FormSelectOption
         isDisabled={false}
         key="Module type"
```
Comment (rednitish) [inline pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap:154]: Can we also consider changing other module instances changes within this PR?

File: pantheon-bundle/frontend/src/app/searchFilter.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ class SearchFilter extends Component<any, any> {
       { value: 'Uploaded date', label: 'Uploaded date', disabled: false },
       { value: 'Title', label: 'Title', disabled: false },
       { value: 'Updated date', label: 'Updated date', disabled: false },
-      { value: 'Module type', label: 'Module type', disabled: false }
+      { value: 'Module type', label: 'Content type', disabled: false }
```
Comment (rednitish) [inline pantheon-bundle/frontend/src/app/searchFilter.tsx:60]: Can we also consider changing other module instances changes within this PR?

**Review by rednitish — CHANGES_REQUESTED** [review]
I have few queries below, let me know, if we have those in separate change list.

File: pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap (hunk 1 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap:157]: that would be a different effort. This ticket I'm working on is weighted as 0.5

File: pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap (hunk 2 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap:154]: changing the key value breaks the sort.

File: pantheon-bundle/frontend/src/app/searchFilter.tsx (hunk 3 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/searchFilter.tsx:60]: changing the key value breaks the sort. We are going to re-do the search page in the near future. For this ticket, the goal is to just update the label.

**Comment by rednitish** [comment]
LGTM

**Review by rednitish — CHANGES_REQUESTED** [review]
LGTM

**Review by rednitish — APPROVED** [review]

**Pull request merged**